app/models/requests/mediated_page.rb

```diff
@@ -42,6 +42,8 @@ def item_limit
   end
 
   def requires_needed_date?
+    return false if origin == 'HOPKINS'
+    return false if origin_location == 'PAGE-MP'
```

Comment:
I thought the same thing at first but didn't really know what that config should be.  We're kind of blacklisting a library and location here so do you think something like `date_needed_library_blacklist` and `date_needed_location_blacklist` would be appropriate?